src/com/google/enterprise/gsafeed/examples/SendFeed.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.enterprise.gsafeed.examples;
+
+import com.google.enterprise.gsafeed.GsaFeedFileSender;
+import com.google.enterprise.gsafeed.Gsafeed;
+import com.google.enterprise.gsafeed.GsafeedHelper;
+import com.google.enterprise.gsafeed.IOHelper;
+
+import java.io.ByteArrayInputStream;
+import java.net.URL;
+import java.nio.charset.Charset;
+
+/**
+ * Example of sending a feed to the GSA.
+ */
+public class SendFeed {
+
+  public static void main(String... args) throws Exception {
+    Charset utf8 = Charset.forName("UTF-8");
+
+    String feedFile = args[0];
```

Comment:
I know this is just a skeleton, but it's easy enough to check args.length and print a usage line.